jax-inference-offloading/examples/rollout.py

```diff
@@ -39,7 +39,7 @@ def main():
   load_format = os.environ.get("VLLM_LOAD_FORMAT", "dummy")
   model_name = os.environ.get("MODEL_NAME", "meta-llama/Llama-3.1-8B-Instruct")
   model_path = os.environ.get("MODEL_PATH", None)
-  model = model_path or model_name
+  model = model_name or model_path
```

Comment:
Are these changes related? rollout.py, requirements.in, example-transfer.sh, oss.dockerfile? If they're not related, please merge them separately. If they are, can you add some explanation?

Reply:
The tokenizer error no longer reproduces when I rerun it today, so it may have been related to the specific checkpoint I downloaded at the time. I’ll revert the change in `rollout.py` for now and keep an eye out in case the issue resurfaces.

Reply:
`oss.dockerfile` and `requirements.in`: Updated to align with the new FlashInfer installation process introduced in vLLM v0.11.0.
`example-transfer.sh` and `rollout.py`: Reverted, since the tokenizer issue no longer reproduces.